### PR TITLE
One single repo

### PR DESCRIPTION
**Pull request by algesten**
This PR removes all git submodules and moves the code into the main webrtc repo.

### CODE REVIEW

File: dtls/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+description = "A pure Rust implementation of DTLS"
+license = "MIT/Apache-2.0"
+documentation = "https://docs.rs/webrtc-dtls"
+homepage = "https://webrtc.rs"
+repository = "https://github.com/webrtc-rs/dtls"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+util = { path = "../util", package = "webrtc-util", default-features = false, features = ["conn"] }
```
Comment (thomaseizinger) [inline dtls/Cargo.toml:15]: I think the all workspace dependencies will need version numbers eventually, once you want to publish one of these.

File: data/codecov.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+codecov:
```
Comment (thomaseizinger) [inline data/codecov.yml:1]: Nit: These can probably go now?

**Review by thomaseizinger — COMMENTED** [review]
Excited for this! Thanks for pushing this forward :)

**Comment by xnorpx** [comment]
looks like history is not there.

https://stackoverflow.com/questions/1759587/how-to-un-submodule-a-git-submodule

**Comment by xnorpx** [comment]
Readme.md needs update

File: webrtc/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+# [minimal-versions]
+# fixes "the trait bound `time::Month: From<u8>` is not satisfied"
+# https://github.com/est31/rcgen/pull/88
+time = "~0.3.1"
+
+[dev-dependencies]
+tokio-test = "0.4.0" # must match the min version of the `tokio` crate above
+env_logger = "0.9.0"
+
+[profile.dev]
```
Comment (xnorpx) [inline webrtc/Cargo.toml:57]: profile should move to workspace to avoid warning

**Comment by xnorpx** [comment]
warning: `turn` (lib test) generated 8 warnings

**Comment by algesten** [comment]
Close in favor of #239

**Pull request closed without merging**